gammapy/modeling/models/spatial.py

```diff
@@ -736,10 +739,11 @@ class GeneralizedGaussianSpatialModel(SpatialModel):
         Shape parameter within (0, 1). Special cases for disk: ->0, Gaussian: 0.5, Laplace:1
         Default is 0.5.
     e : `float`
-        Eccentricity (:math:`0< e< 1`).
+        Eccentricity (:math:`0<= e< =1`).
         Default is 0.
     phi : `~astropy.coordinates.Angle`
-        Rotation angle :math:`\phi`: of the major semiaxis.
+        Rotation angle :math:`\phi`: of the major semiaxis (:math:`-20<=phi<=200`).
+        phi is wrapped between [0, 180] with the larger range enabled to enable fits for phi close to 0
```

Comment:
"with the larger range enabled to enable fits for phi close to 0"  : I think this is an implementation detail.